tests/test_api_requestor.py

```diff
@@ -12,7 +12,7 @@
 from stripe import six
 from stripe.stripe_response import StripeResponse
 
-from six.moves.urllib.parse import urlsplit
+from stripe.six.moves.urllib.parse import urlsplit
```

Comment:
This was an actual bug in tests! The previous version used the `six` module installed in the venv, not the one bundled with stripe-python. Now tox creates a new venv for each test run and `six` is not installed in that venv, so this started failing.